test_communication/test/test_publisher_subscriber.py.in

```diff
@@ -9,9 +10,11 @@ from launch.launcher import DefaultLauncher
 
 
 def test_publisher_subscriber():
+    now = datetime.datetime.now()
+    namespace = '/test_time_{0}_{1}_{2}'.format(now.hour, now.minute, now.second)
```

Comment:
Since you are only interested in the time I would suggest to use `time` instead of `datetime`.